frontend/src/pages/landingPage/_landingPage.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../components/Buttons/Buttons";
+@import "../../components/Basics/breakpoints";
+@import "../../components/Basics/typography";
+
+.image-parent {
+  z-index: 1;
+  margin-top: -10%;
+  @include media-max("smtablet") {
+    margin-top: 0;
+  }
+
+  position: relative;
+  background-image: url("./assets/images/svgs/group-background-pattern.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 100%;
+}
+
+.group-image {
```

Comment:
One more thing: the class names are very non-specific. Since we have changed to loading all styles at once, it would cause a lot of trouble if we accidentally reuse a class name. Let's rename some of them to, for example, "landing-image" or "landing-mission-title", and so forth.